resolver/src/main/java/io/netty/resolver/SimpleNameResolver.java

```diff
@@ -168,12 +156,80 @@ public final boolean isResolved(SocketAddress address) {
         }
     }
 
+    @Override
+    public final Future<List<T>> resolveAll(String inetHost, int inetPort) {
+        return resolveAll(InetSocketAddress.createUnresolved(checkNotNull(inetHost, "inetHost"), inetPort));
+    }
+
+    @Override
+    public Future<List<T>> resolveAll(String inetHost, int inetPort, Promise<List<T>> promise) {
+        return resolveAll(InetSocketAddress.createUnresolved(checkNotNull(inetHost, "inetHost"), inetPort), promise);
+    }
+
+    @Override
+    public final Future<List<T>> resolveAll(SocketAddress address) {
+        if (!isSupported(checkNotNull(address, "address"))) {
+            // Address type not supported by the resolver
+            return executor().newFailedFuture(new UnsupportedAddressTypeException());
```

Comment:
Should we include the address in the exception message so it is easier to debug ?